kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -166,6 +167,49 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  test("create table and set properties") {
```

Comment:
Can you add one more test case here:
1. First commit is written with valid table properties
2. The second commit does not specify any table properties ---> the original properties should be retained.